admin_manual/maintenance/upgrade.rst

```diff
@@ -36,14 +36,17 @@ After the upgrade is complete re-enable any which are compatible with the new re
 Upgrade Options
 ---------------
 
-There are three ways to upgrade your ownCloud server:
+There are three ways to upgrade your ownCloud server.
+The first — *and recommended* — way is to perform a :doc:`manual upgrade <manual_upgrade>`, using `the latest ownCloud release <owncloud.org/install/>`_.
+The second way is to use your distribution's :doc:`package manager <package_upgrade>`, in conjunction with our official ownCloud repositories. 
+However, this approach should not be used unattended nor in clustered setups.
 
-#. Using the :doc:`Updater App <update>` (Server Edition only). This is recommended for shared hosters and for users who want an easy way to track different release channels. It is *not available* and *not supported* on the Enterprise edition. This is the recommended method.
-#. Using your :doc:`Linux package manager <package_upgrade>` with our official ownCloud repositories. 
-#. :doc:`Manually upgrading <manual_upgrade>` with the latest ownCloud release from `owncloud.org/install/`_. 
+The third way is to use the :doc:`Updater App <update>`. 
+This is only available with community edition and cannot be used with ownCloud Enterprise Edition.
+It is recommended for shared hosting environments and for users who want an easy way to track different release channels. 
 
 .. note::
-   Please remember that in scale-out deployments (where your database is sitting somewehere else, not on the same machine as the webserver) using the package manager is going to install all the related dependencies, unless the ``owncloud-enterprise-files`` (for the Enterprise edition) or the ``owncloud-files`` (for the Community edition) package is being used.
+   Please remember that in scale-out deployments (where your database is sitting somewehere else, not on the same machine as the webserver) using the package manager is going to install all the related dependencies, unless the ``owncloud-enterprise-files`` (for the Enterprise Edition) or the ``owncloud-files`` (for the Community edition) package is being used.
```

Comment:
1. A scale-out deployment is not defined by a database sitting on a different host. It's rather that you have load balancers, multiple application servers and/or clustered database and/or redis setups. I'm not the best reference for this but quite sure that this would be a better description.
2. I'm also quite sure that package managers should not be used in such scenarios.
3. There are other things to be considered that are not mentioned here. I recommend having a separate location to document upgrading scale-out deployments.
4. Also the package manager won't install any related dependencies anymore as there is only the "owncloud-files" package. This has been changed some time ago to avoid potentially breaking other services by forcefully installing dependencies (e.g. PHP extensions, database, etc.).

Reply:
@pmaier1, regarding #3 above, who's the best to talk to, to gather this information?